src/validator.directive.js

```diff
@@ -33,6 +33,7 @@
         messageElem.css('display', '');
       } else {
         messageElem.css('display', 'none');
+        messageElem.html('');
```

Comment:
Fixes the error message not cleaning up when the validation passes for a single input.